bioio_ome_zarr/tests/test_reader.py

```diff
@@ -179,3 +205,31 @@ def test_ome_zarr_reader(
         expected_physical_pixel_sizes=expected_physical_pixel_sizes,
         expected_metadata_type=dict,
     )
+
+
+cloud_path = (
+    "https://animatedcell-test-data.s3.us-west-2.amazonaws.com/variance/35.zarr"
```

Comment:
How confident are we in the stability of this specific file?  Maybe clone it somewhere in a directory explicitly marked 'test resource'?  Maybe it is a fixed file, I don't know.

Reply:
This is one of the files from the variance dataset. @toloudis do you have an opinion on this? I have been trying to increase the use of publically available data in our tests.

Reply:
I do think `animatedcell-test-data` should be considered volatile and not necessarily reserved for permanent unit tests, and we should either move the file somewhere more permanent or use something else.

Reply:
Do we have some publically released zarrs that could be used for this purpose?

Reply:
Most things in our [public quilt](https://open.quiltdata.com/b/allencell/tree/aics/) are intended to stay there long term. For ZARRs I suggest looking at https://open.quiltdata.com/b/allencell/tree/aics/nuc-morph-dataset/. Most of the ZARRs there have 5 resolution levels. You'll need a URL that points directly to S3, not quilt, so look at my test in this repo for an example.

Reply:
I am not sure definitively but see #24 for some hints

Reply:
updated in [7194507](https://github.com/bioio-devs/bioio-ome-zarr/pull/23/commits/7194507fbe0c5f51280b3d0f72502259716380f5)